subsys/bluetooth/host/conn.c

```diff
@@ -2810,6 +2810,10 @@ struct net_buf *bt_conn_create_frag_timeout(size_t reserve, k_timeout_t timeout)
 #if defined(CONFIG_BT_SMP) || defined(CONFIG_BT_BREDR)
 int bt_conn_auth_cb_register(const struct bt_conn_auth_cb *cb)
```

Comment:
Can you add a line to the function's docstring documenting the new behavior ?

Reply:
Done, please rereview

Reply:
can you also do the same (impl + doc) for the `unregister` function ? Also I don't understand, can we update the callbacks at all ? it looks like we'll get the `EALREADY` if they already exist and we want to modify them.

Reply:
my bad I read `bt_conn_auth_info_cb_unregister` too fast.